// [[Reason: TSC adds type reference directives.]] ////

//// [tests/cases/compiler/typeReferenceDirectives2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -3,19 +3,4 @@
 //// [/app.d.ts]
 interface A {
     x: $;
 }
-
-/// [Errors] ////
-
-/app.ts(2,8): error TS9024: Declaration emit for this expression requires adding a type reference directive to 'lib' with --isolatedDeclarations.
-
-
-==== /app.ts (1 errors) ====
-    interface A {
-        x: $
-           ~
-!!! error TS9024: Declaration emit for this expression requires adding a type reference directive to 'lib' with --isolatedDeclarations.
-    }
-==== /types/lib/index.d.ts (0 errors) ====
-    interface $ { x }
-    
\ No newline at end of file
